packages/client/src/app/pages/content-views/section-view/section-view.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit } from '@angular/core';
+import { ContentService } from '../../../services/content';
+import { ContentModel, SectionInfo } from '@pulp-fiction/models/content';
+import { Section } from '@pulp-fiction/models/sections';
+import { ActivatedRoute, Router } from '@angular/router';
+import * as v from 'voca';
+
+@Component({
+    selector: 'section-view',
+    templateUrl: './section-view.component.html',
+    styleUrls: ['./section-view.component.less']
+})
+export class SectionViewComponent implements OnInit {
+    sections: SectionInfo[];
+    thisSection: Section;
+    loading = false;
+
+    currIndex: number;
+    indexNext: number;
+    indexPrev: number;
+
+    constructor(private contentService: ContentService, private route: ActivatedRoute, private router: Router) {
+        this.fetchData();
+    }
+
+    ngOnInit(): void {}
+
+    private fetchData() {
+        this.loading = true;
+        this.route.paramMap.subscribe(params => {
```

Comment:
Can you use the `paramMap.snapshot` here? Or do you actually need a persistent subscription?